content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
+**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
+the interface you access using your browser.
+
+Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!
```

Comment:
The english here seems a bit off. I suggest instead using " we are unsure if we covered everything that is new." It has to words that break the flow right now.

"We hope so." should be changed to "We sure hope we have!" and the final sentence here should be changed to "Grab some popcorn and get ready to dive in!"